Sources/UnleashProxyClientSwift/UnleashProxyClientSwift.swift

```diff
@@ -43,8 +38,8 @@ public class UnleashClientBase {
             fatalError("Invalid Unleash URL: \(unleashUrl)")
         }
 
-        self.context["appName"] = appName
-        self.context["environment"] = environment
+        self.context = Context(appName: appName, environment: environment, properties: [:])
```

Comment:
Do you have to pass properties here or could you do it like this? 
```swift
        self.context = Context(appName: appName, environment: environment)
```

Reply:
good catch. not needed